catalog/ui/search-ui/standard/src/main/doc/_contents/help-contents.adoc

```diff
@@ -0,0 +1,373 @@
+
+|Summary button
+|Opens the Summary tab, which provides a high level overview of the result set.
+
+|Id
+|The record's unique identifier.
+
+|Source Id
+|Where the metadata was retrieved from, which could be the local provider or a federated source.
+
+|Title
+|The title of the record is displayed in white font.
+
+|Thumbnail
+|Depicts a reduced size image of the original artifact for the current record, if available.
+
+|Resource URI
+|Identifies the stored resource within the server.
+
+|Created time
+|When the record was created.
+
+|Metacard Content Type version
+|The version of the metadata associated with the record.
+
+|Metacard Type
+|The type of metacard associated with the record.
+
+|Metacard Content Type
+|The type of the metadata associated with the record.
+
+|Resource size
+|The size of the resource, if available.
+
+|Modified
+|Time since the record was last modified.
+
+|Download
+|When applicable, a download link for the product associated with the record is displayed. The size
+ of the product is also displayed, if available. If the size is not available, N/A is displayed.
+
+|Metadata
+|Shows a representation of the metadata XML, if available.
+|===
+
+image:record-details.png[Record details]
+
+=== Actions
+
+Depending on the contents of the metacard, various actions will be available to perform on the
+metadata.
+
+Troubleshooting: if no actions are available, ensure IP address is configured correctly under
+global configuration in Admin Console.
+
+image:record-actions.png[Actions]
+
+==== Save a Search
+
+Saved searches are search criteria that are created and saved by a user. Each saved search has a
+name that was defined by the user, and the search can be executed at a later time or be scheduled
+for execution. Saved records (metacards) that the user elected to save for future use are returned
+as part of a search. These queries can be saved to a <<Workspaces, workspace>>, which is a
+collection of searches and metacards created by a user. Complete the following procedure to create
+a saved search.
+
+. Select the Search tab at the bottom of the left pane.
+. Use the fields provided to define the <<_search_criteria>> for the query to be saved.
+. Select the *Save* button. The Select Workspace pane opens.
+. Type a name for the query in the *ENTER NAME FOR SEARCH* field.
+. Select a workspace in which to save the query, or create a workspace by typing a title for the
+  new workspace in the *New Workspace* field.
+  image:search-save.png[New workspace]
+
+. Select the Save button.
+
+
+[NOTE]
+====
+The size of the product is based on the value in the associated metacard's resource-size attribute.
+This is defined when the metacard was originally created and may or may not be accurate. Often it
+will be set to N/A, indicating that the size is unknown or not applicable.
+
+However, if the administrator has enabled caching on {branding} and has installed the
```

Comment:
This is a written list. Modify to this: `...caching on {branding}, has installed the...`